packages/terra-application/src/application-navigation/private/drawer-menu/_DrawerMenuUser.jsx

```diff
@@ -42,6 +53,19 @@ const DrawerMenuUser = ({ userConfig, variant }) => {
       <div className={cx('info-container')}>
         <div aria-hidden className={cx('name')}>{userConfig.name}</div>
         {userConfig.detail ? <div className={cx('detail')}>{userConfig.detail}</div> : null}
+        { userActionConfig && userActionConfig.text && userActionConfig.userActionCallback && (
+        <button
```

Comment:
Why are we not using terra-button here?

Reply:
UX decision.

Reply:
I think it is because terra-button has some styling differences when it is rendered within Drawer Menu. ( terra-button border styles doesn't fit well with blue background of drawer-menu )  @neilpfeiffer correct me if I am wrong.

Reply:
Correct. In the drawer the button needs specific styling so a unique button is better here.